packages/worker/src/api/controllers/system/environment.ts

```diff
@@ -1,6 +1,25 @@
 import { Ctx } from "@budibase/types"
 import env from "../../../environment"
 import { env as coreEnv } from "@budibase/backend-core"
+import nodeFetch from "node-fetch"
+
+let sqsAvailable: boolean
+async function isSqsAvailable() {
+  if (sqsAvailable !== undefined) {
+    return sqsAvailable
+  }
+
+  try {
+    await nodeFetch(coreEnv.COUCH_DB_SQL_URL, {
```

Comment:
Will SQS be always available per instance, and not per tenant? And can we have a DDoS issue on worker restarts, if there are many requests and a slow sqs instance?

Reply:
It's my understanding that SQS will be a global thing. We're baking it in to the CouchDB image for self-hosters, unsure what we're doing in Budicloud.

I don't think this poses a DDoS concern. The request to `/` is very fast (it returns a few metrics) and this request is sent once per lifetime of a pod.

Reply:
Yeah SQS is global, the whole self host cluster will need SQS support. I don't think this poses a threat of DDoS either as it only runs in self host, and in most installations there is max 1-2 worker pods, if they get stuck in a restart look its still only a few hundred requests max.